libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-wrapper.component.html

```diff
@@ -8,9 +8,10 @@
 <div
   class="sky-ag-grid"
   [ngClass]="{
-    'sky-ag-grid-top-scrollbar':
-      agGrid?.gridOptions?.context &&
-      agGrid?.gridOptions?.context['enableTopScroll']
+    'sky-ag-grid-top-scrollbar': enableTopScroll,
+    'ag-theme-sky-default': isDefaultTheme,
+    'ag-theme-sky-modern-light': isModernLightTheme,
+    'ag-theme-sky-modern-dark': isModernDarkTheme
```

Comment:
Could we create one property named `skyThemeClassName` (or something similar) that is constructed by the `SkyThemeService.settingsChange` subscription? This will reduce the number of public properties on the component.

Reply:
Moved top-scrollbar out of `ngClass` and consolidated themes to one variable.